spec/features/authentication_spec.rb

```diff
@@ -32,6 +32,46 @@
         expect(page).to have_content(I18n.t('sessions.create.failure'))
       end
     end
+
+    context 'with no authentication token' do
+      let(:request_for_comment) { create(:rfc_with_comment, user: user) }
+
+      it 'denies access to the request for comment' do
+        visit(request_for_comment_path(request_for_comment))
+        expect(page).to have_content(I18n.t('application.not_authorized'))
```

Comment:
This very much replicates implementation. We should check for expected content.
Similar below.

Reply:
Okay, I got it (using I18n vs a string representation). Besides considering your comment for a refactoring of all specs, I would vote to leave it this way for now (to ensure all occurrences are consistent).